code/_compile_options.dm

```diff
@@ -109,9 +109,9 @@
 #endif
 
 #if !defined(CBT) && !defined(SPACEMAN_DMM)
-#warn Building with Dream Maker is no longer supported and will result in errors.
-#warn In order to build, run BUILD.bat in the root directory.
```

Comment:
Makes this an error message too, people should know this exists

Reply:
If they're at the point of using the scripts directly, they probably don't need an error telling them about it. For the general user, I believe the intent is to push them entirely towards vscode.